src/amo/pages/SearchTools/index.js

```diff
@@ -3,85 +3,50 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
-import Search from 'amo/components/Search';
-import HeadLinks from 'amo/components/HeadLinks';
-import HeadMetaTags from 'amo/components/HeadMetaTags';
-import Page from 'amo/components/Page';
 import { ADDON_TYPE_EXTENSION, SEARCH_SORT_TOP_RATED } from 'amo/constants';
 import translate from 'amo/i18n/translate';
-import { convertFiltersToQueryParams } from 'amo/searchUtils';
 import { makeQueryString } from 'amo/api';
 import { sendServerRedirect } from 'amo/reducers/redirectTo';
-import { getCategoryResultsQuery } from 'amo/utils/categories';
+import { getCategoryResultsPathname } from 'amo/utils/categories';
 import type { AppState } from 'amo/store';
 import type { DispatchFunc } from 'amo/types/redux';
-import type { I18nType } from 'amo/types/i18n';
-import type { SearchFilters } from 'amo/components/AutoSearchInput';
 
 type Props = {|
-  filters: SearchFilters,
-|};
-
-type InternalProps = {|
-  ...Props,
   clientApp: string,
   dispatch: DispatchFunc,
-  i18n: I18nType,
   lang: string,
 |};
 
-export class SearchToolsBase extends React.Component<InternalProps> {
-  constructor(props: InternalProps) {
+export class SearchToolsBase extends React.Component<Props> {
+  constructor(props: Props) {
     super(props);
 
     const { clientApp, dispatch, lang } = props;
 
-    const queryString = makeQueryString(
-      getCategoryResultsQuery({
-        addonType: ADDON_TYPE_EXTENSION,
-        slug: 'search-tools',
-      }),
-    );
+    const pathname = getCategoryResultsPathname({
+      addonType: ADDON_TYPE_EXTENSION,
+      slug: 'search-tools',
+    });
 
     dispatch(
       sendServerRedirect({
         status: 301,
-        url: `/${lang}/${clientApp}/search/${queryString}`,
+        url: `/${lang}/${clientApp}${pathname}${makeQueryString({
+          sort: SEARCH_SORT_TOP_RATED,
```

Comment:
mm it isn't the same thing as before, right? I don't see a `sort` filter in the previous code?

Reply:
It was in `mapStateToProps` in the old code, but looking at the current behaviour of the site it isn't being picked up, so I guess we can remove it.